### PR TITLE
[azeventhubs+azservicebus] stress test updates for new matrix format, and fix for a failing test

### LINKED ISSUE #19427: [stress] azeventhubs and azservicebus need to upgrade to the new matrix scenario format
The stress deployer now allows for matrix-based configuraiton (rather than simplistic 'values.yaml' version before), which makes it simpler to split things apart and add parameters to each scenario, rather than having to hard code everything.

We need to do some updating for both SB and EH's stress tests to use it, as it's the new format going forward.

### PR DESCRIPTION
**Pull request by richardpark-msft**
- Failing test fix: Increasing default batch timeout (in test) to 60s, rather than 1s. We're lazy initialized, and 1s doesn't seem like it's always enough time to get the connection and link off the ground.
- Updating all the yamls and stress-argy-bargy for the new (and final) format where we use the matrix generator. 

Fixes #19427.

### CODE REVIEW

File: sdk/messaging/azeventhubs/internal/eh/stress/.gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ charts
 newstress
 oldstress
 *.log
+generatedValues.yaml
```
Comment (jhendrixMSFT) [inline sdk/messaging/azeventhubs/internal/eh/stress/.gitignore:8]: nit: Missing trailing new line

**Review by jhendrixMSFT — APPROVED** [review]
Looks like there are several missing new-line chars, up to you if the red circles hurt your head.

**Comment by richardpark-msft** [comment]
> Looks like there are several missing new-line chars, up to you if the red circles hurt your head.

I need to fix up my editor settings or something. It's not intentional, will fix.

File: sdk/messaging/azeventhubs/internal/eh/stress/.gitignore (hunk 1 above)
Comment (richardpark-msft) [reply sdk/messaging/azeventhubs/internal/eh/stress/.gitignore]: Newline added.

**Pull request merged**